tests/test_detector.py

```diff
@@ -53,20 +44,42 @@ def test_instantiation(self):
         self.assertIsInstance(self.method, TUVMethod)
 
     def test_get_channels(self):
-        channel_dict = self.method.channel_dict
-        self.assertEqual(len(channel_dict), 2)
-        self.assertEqual(channel_dict["Channel1"]["Type"], "Single")
-        self.assertEqual(channel_dict["Channel1"]["Wavelength"], "214")
+        channels = self.method.channels
+        self.assertEqual(len(channels), 1)
+        self.assertEqual(channels[0].wavelength, "214")
+
+    def test_set_channels(self):
+        self.method.channels = [TUVChannel(wavelength="400")]
+        channels = self.method.channels
+        self.assertEqual(len(channels), 1)
+        self.assertEqual(channels[0].wavelength, "400")
+
+    def test_set_channels_add(self):
+        self.assertEqual(self.method.channels[0].datamode, "SingleMode_1A")
+        self.assertEqual(self.method.channels[0].datarate, "SingleDataRate_20A")
+        self.assertEqual(self.method.channels[0].timeconstant, "0.1")
+        self.method.channels = [
+            TUVChannel(wavelength="400"),
+            TUVChannel(wavelength="500"),
+        ]
+        channels = self.method.channels
```

Comment:
Should we also check the xml? That is what really matters, right? Either with tags, or just the values.

Reply:
Ill do a basic, assertIsIn rather tahn looking through trees for now